engine-comparison/runner.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+. metadata.cfg
+. parameters.cfg
+. fengine.cfg
+
+BINARY=${BENCHMARK}-${FUZZING_ENGINE}
+mkdir -p corpus
+chmod 750 $BINARY
+
+if [[ FUZZING_ENGINE == "afl" ]]; then
+  chmod 750 afl-fuzz
+  BINARY="afl-fuzz $BINARY"
+fi
+
+./$BINARY $BINARY_RUNTIME_OPTIONS # -workers=$JOBS -jobs=$JOBS -artifact_prefix=corpus
+
+mkdir -p results
+while [[ ! -e results/complete.txt ]] : do
+  sleep 12
+  # TODO: create a copy fuzz-0.log here; the copy can be read to reduce errors
```

Comment:
What about the other `fuzz-#.log` files that get created?  What do you mean by "the copy can be read to reduce errors"?